examples/README.rst

```diff
@@ -0,0 +1 @@
+The software in this directory (Cirq/examples/) is not included in the packaged library. Anyone is free to copy, modify, or publish this software.
```

Comment:
"this example code" instead of "this software". And add "use" in the list of things they're allowed to do.

Reply:
Done.